From 45a1bc294cf15af87fb3a387a1b4cbadb15d8b2d Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:42:38 -0400
Subject: [alsa] ice1712: Replacing hex with #defines

Message-id: <140972823573-442-patchset-jkysela@redhat.com>
Patchwork-id: 91131
O-Subject: [RHEL 7.1 PATCH 441/442] ALSA: ice1712: Replacing hex with #defines
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Adds to the readability of the ice1712 driver.

Signed-off-by: Konstantinos Tsimpoukas <kostaslinuxxx@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 890b13a308b9df78ad05fc57eb440b32399be35e)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/ice1712/ice1712.c b/sound/pci/ice1712/ice1712.c
index 87f7fc4..206ed2c 100644
--- a/sound/pci/ice1712/ice1712.c
+++ b/sound/pci/ice1712/ice1712.c
@@ -2528,7 +2528,7 @@ static int snd_ice1712_free(struct snd_ice1712 *ice)
  if (!ice->port)
   goto __hw_end;
  /* mask all interrupts */
- outb(0xc0, ICEMT(ice, IRQ));
+ outb(ICE1712_MULTI_CAPTURE | ICE1712_MULTI_PLAYBACK, ICEMT(ice, IRQ));
  outb(0xff, ICEREG(ice, IRQMASK));
  /* --- */
 __hw_end:
-- 
1.7.1